hazelcast-hibernate/hazelcast-hibernate3/src/test/java/com/hazelcast/hibernate/entity/DummyEntityNonStrictRW.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.hibernate.entity;
+
+import java.util.Date;
+import java.util.Set;
+
+public class DummyEntityNonStrictRW {
```

Comment:
What does non strict mean? PLease add some documentation. And what is rw thing? Read write? Do you have none rw version? If not; I would get rid of it since most entities are updatable in my experience.